fairlearn/widget/fairness_dashboard.py

```diff
@@ -28,26 +28,28 @@
 
 
 class FairlearnDashboard(object):
-    """The dashboard class, wraps the dashboard component.
+    r"""The dashboard class, wraps the dashboard component.
 
-    :param sensitive_features: A matrix of feature vector examples (# examples x # features),
+    Parameters
+    ----------
+    sensitive_features : {:class:`numpy.ndarray`, list[][], :class:`pandas.DataFrame`, :class:`pandas.Series`}  # noqa: E501
```

Comment:
`# noqa` [renders!](https://1786-133444044-gh.circle-artifacts.com/0/docs/_build/html/master/api_reference/fairlearn.widget.html)
also, please get rid of enclosing `{}` around listing of types

Reply:
Saw this `{}` elsewhere in the repo and assumed that's how we want to express this. I suppose that's tricky because I can't spread it across multiple lines (renders the wrong way). Perhaps if I don't use `:class:` it will be short enough. Will try this...

Reply:
removed